Pendulum/Views/Sheets/AddEventSheet.swift

```diff
@@ -179,49 +181,67 @@ struct AddEventSheet: View {
+                                    }
+                                    .offset(y: 8)
                                 }
                             }
-                            HStack {
+                            HStack(alignment: .top) {
                                 Image(systemName: "drop")
                                     .foregroundColor(.secondary)
-                                HStack {
-                                    TextField(priorWrittenEvent?.ink ?? "Ink", text: $ink)
-                                        .focused($isInkFieldActive)
-                                    if !inkSuggestions.isEmpty {
-                                        Button(action: {
-                                            presentSuggestionSheetFor = TextOptions(text: $ink, options: inkSuggestions, title: "Choose an Ink")
-                                        }) {
-                                            Image(systemName: "ellipsis")
+                                    .offset(y: 1)
+                                    .background {
+                                        GeometryReader { geo in
+                                            Color.clear.preference(key: Self.IconWidthPreferenceKey.self, value: geo.size.width)
                                         }
                                     }
+                                    .frame(width: iconWidth)
+                                TextField(priorWrittenEvent?.ink ?? "Ink", text: $ink, axis: .vertical)
+                                    .focused($isInkFieldActive)
+                                if !inkSuggestions.isEmpty {
+                                    Button(action: {
+                                        presentSuggestionSheetFor = TextOptions(text: $ink, options: inkSuggestions, title: "Choose Inks")
+                                    }) {
+                                        Image(systemName: "ellipsis")
+                                    }
+                                    .offset(y: 8)
                                 }
                             }
-                            HStack {
+                            HStack(alignment: .top) {
                                 Image(systemName: "doc.plaintext")
                                     .foregroundColor(.secondary)
-                                HStack {
-                                    TextField(priorWrittenEvent?.paper ?? "Paper", text: $paper)
-                                        .focused($isPaperFieldActive)
-                                    if !paperSuggestions.isEmpty {
-                                        Button(action: {
-                                            presentSuggestionSheetFor = TextOptions(text: $paper, options: paperSuggestions, title: "Choose a Paper")
-                                        }) {
-                                            Image(systemName: "ellipsis")
+                                    .offset(y: 1)
```

Comment:
These offsets are going to end up killing us 😂

Reply:
Yes, yes they are 😭